cmd/neofs-cli/modules/object.go

```diff
@@ -145,6 +147,8 @@ func initObjectPutCmd() {
 	flags.Bool("disable-timestamp", false, "Do not set well-known timestamp attribute")
 	flags.Uint64VarP(&putExpiredOn, putExpiresOnFlag, "e", 0, "Last epoch in the life of the object")
 	flags.Bool(noProgressFlag, false, "Do not show progress bar")
+
+	flags.String(notificationFlag, "", "Object notification in the form of *epoch*:*topic*; '-' topic means using default")
```

Comment:
Is `-` reserved on the API level? Seems like empty topic would agree better on what is happening in the code.

Reply:
> Is - reserved on the API level?

No. Just an explicit sign that we want to use the node's default topic.
/cc @alexvanin